### ISSUE TITLE
LoG(img) question

### ISSUE THREAD

**Issue by 12michi34** [Black-Phoenix/Ai-Path-Tracer-Denoiser#2]
Hi,
Just got a question regarding LoG(img).
The current code does 
```
	weight_np = np.repeat(weight_np, img.shape[1], axis=1)
	weight_np = np.repeat(weight_np, img.shape[0], axis=0)
```
which I assume tries to replicate the filer along the channel and minibatch axis.

but looking at  
https://pytorch.org/docs/stable/generated/torch.nn.functional.conv2d.html
the function wants 
(out_channels, in_channels, H, W) 
shape.

What I mean is that in case you have a (10,3,128,128) input tensor you would build a 
weight of shape (10,3,3,3) and that gives you an output image of (10,10,128,128) because the 1st dimension of the weight 
defines the out_channels (see docs). 
Is that really the intention should you not get a (10,3,128,128) tensor out in the end ?
In that case I think a weight of shape (3,3,3,3) would be required.

Also when I try to run the code I get a perfect greyscale image for the edge-detected image but your readme has a green/yellow color.
Maybe I have a bug on my side but wanted to check if that image in the readme section of the project was captured using the python code provided ?

Cheers

**Comment by Black-Phoenix** [comment]
Reg the code question, I have honestly not looked at this code in years, so I will need some time to get up to speed on what it wanted to do. I will try to look into this later this month when I have a few more cycles. Regarding the greyscale imgs, all the imgs in the readme were captured using the code in the repo, and are expected to be in color if the 1spp has color.
